### ISSUE TITLE
Reference-based evaluation doesn't detect query

### ISSUE THREAD

**Issue by desmodus1984** [Maggi-Chen/Inspector#39]
Hi,

I tried the reference-based evaluation, but it failed

The code
# Reference-based only evaluation
```
inspector.py -c /home/pinn+refs31-scaff/final_assembly.fasta -r emptyfile \
        --ref /home/data/genomes/HydLep.fx.fasta \
        -o Scaff-JC/ -t 64
```

And the error:

```
Inspector starting... 13/09/2025 16:39:24
Start Assembly evaluation with contigs: ['home/pinn+refs31-scaff/final_assembly.fasta']
Warning: cannot open input file "emptyfile". Removed from list.
Error: No valida input read file. Abort.

```

Any reason why it failed?
I used the code as stated on the website.
I was able to use seqkit and head on the final_assembly.fasta file.
